src/index.ts

```diff
@@ -1,3 +1,6 @@
+// Import modules
+import ngrok = require('ngrok');
```

Comment:
Change to `import ngrok from 'ngrok';`

Reply:
Got it! Linking the StackOverflow post you sent me to keep up documentation

https://stackoverflow.com/questions/37651495/difference-between-import-from-and-import-require-in-typescript